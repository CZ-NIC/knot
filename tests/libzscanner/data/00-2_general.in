$ORIGIN zone.
$TTL  4
;        
@ NS ns.
        
@ 1 TXT text 
