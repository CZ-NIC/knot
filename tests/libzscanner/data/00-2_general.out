OWNER=047A6F6E6500
CLASS=0001
RRTTL=00000004
RTYPE=0002
RDATA=026E7300
------
OWNER=047A6F6E6500
CLASS=0001
RRTTL=00000001
RTYPE=0010
RDATA=0474657874
------
